test/integration/apiserver/cel/authorizerselector/helper.go

```diff
@@ -183,8 +183,8 @@ func RunAuthzSelectorsLibraryTests(t *testing.T, featureEnabled bool) {
 				return err
 			},
 			// authorizer is not available to resource APIs
-			expectErrorsWhenEnabled:  []*regexp.Regexp{regexp.MustCompile(`spec\.devices\.requests\[0\]\.selectors\[0\].cel\.expression:.*undeclared reference to 'authorizer'`)},
-			expectErrorsWhenDisabled: []*regexp.Regexp{regexp.MustCompile(`spec\.devices\.requests\[0\]\.selectors\[0\].cel\.expression:.*undeclared reference to 'authorizer'`)},
+			expectErrorsWhenEnabled:  []*regexp.Regexp{regexp.MustCompile(`spec\.devices\.requests\[0\]\.exactly\.selectors\[0\].cel\.expression:.*undeclared reference to 'authorizer'`)},
+			expectErrorsWhenDisabled: []*regexp.Regexp{regexp.MustCompile(`spec\.devices\.requests\[0\]\.exactly\.selectors\[0\].cel\.expression:.*undeclared reference to 'authorizer'`)},
```

Comment:
@pohly I am a bit uncertain about this test. It currently passes, but there are a few things about it that I don't quite understand. It uses the `v1beta1` API and updating it to use the `v1beta2` API doesn't work. And this is even though the `runtime-config` for this test seems to only enable the `v1alpha3` API (line 49 of this file): 

```
	// Start the server with the desired feature enablement
	server, err := apiservertesting.StartTestServer(t, nil, []string{
		fmt.Sprintf("--feature-gates=AuthorizeNodeWithSelectors=%v,AuthorizeWithSelectors=%v", featureEnabled, featureEnabled),
		"--runtime-config=resource.k8s.io/v1alpha3=true",
```

Reply:
Changing this to use the `v1beta2` API was caused by another issue. I've updated it now and it should pass.